src/txn_builder.rs

```diff
@@ -718,10 +721,10 @@ pub struct TransactionHistoryEntry<L: Ledger> {
     pub time: DateTime<Local>,
     pub asset: AssetCode,
     pub kind: TransactionKind<L>,
-    // If we sent this transaction, `sender` records the address of the spending key used to submit
-    // it. If we received this transaction from someone else, we may not know who the sender is and
-    // this field may be None.
-    pub sender: Option<UserAddress>,
+    // If we sent this transaction, `senders` records the addresses of the spending keys used to
+    // submit it. If we received this transaction from someone else, we may not know who the senders
+    // are and this field may be None.
```

Comment:
This comment is out of sync. I actually kind of like making it an `Option<Vec<UserAddress>>` and continuing to use `None` for "unknown", but i think since we never build a transaction with empty `senders` ourselves, it is also fine to use `vec![]` for "unknown", in which case just update the comment.